libs/navigation/driver/magento/src/transformers/navigation-transformer.ts

```diff
@@ -40,7 +40,7 @@ export class DaffMagentoNavigationTransformerService implements DaffNavigationTr
       id: breadcrumb.category_uid,
       name: breadcrumb.category_name,
       level: breadcrumb.category_level,
-      url: breadcrumb.category_url_key,
+      url: `/${breadcrumb.category_url_path}`,
```

Comment:
While maybe not in scope, this is missing suffix right?

Reply:
Yeah and it's probably in scope as well, I'll add it

Reply:
added and also added the category breadcrumbs